src/constants/Image.js

```diff
@@ -0,0 +1,22 @@
+/* global Images */
+
+import momo from "../resources/images/global/momo.png";
+import salad from "../resources/images/global/salad.png";
+import user from "../resources/images/global/user.png";
+import sandwich from "../resources/images/global/sandwich.png";
+
+/* icons */
+
+import bookmark from "../resources/images/icons/bookmark.png";
```

Comment:
Icons can be retrieved from Figma as SVG and components can be created for them. You can check out this [PR](https://github.com/itsnporg/blog-itsnp-frontend/pull/10/files#diff-93aa9ea357a87a6cd8511b0e91cbe1685f15d7ebc730e61273ae18d484ae8cdd) for more information.